pkg/login/ldap.go

```diff
@@ -278,18 +278,27 @@ func (a *ldapAuther) searchForUser(username string) (*LdapUserInfo, error) {
 	var err error
 
 	for _, searchBase := range a.server.SearchBaseDNs {
+		attributes := make([]string, 0)
+
+		appendIfNotEmpty := func(slice []string, attr string) []string {
```

Comment:
maybe we can move this to a package func or out of the loop (just looks a bit strang to define this util function in the loop :)

Reply:
Could be a func with an `...string` argument.
ex https://play.golang.org/p/deln_a2skzU